packages/perseus-editor/src/widgets/free-response-editor.tsx

```diff
@@ -0,0 +1,46 @@
+import {freeResponseLogic} from "@khanacademy/perseus-core";
+import {LabeledTextField} from "@khanacademy/wonder-blocks-form";
+import {spacing} from "@khanacademy/wonder-blocks-tokens";
+import * as React from "react";
+
+import type {PerseusFreeResponseWidgetOptions} from "@khanacademy/perseus-core";
+
+type Props = PerseusFreeResponseWidgetOptions & {
+    onChange: (arg1: {question?: Props["question"]}) => void;
```

Comment:
I think the type of the onChange arg should be `Partial<PerseusFreeResponseWidgetOptions>`.